core-bundle/src/Routing/ResponseContext/JsonLd/JsonLdManager.php

```diff
@@ -70,7 +70,16 @@ public function collectFinalScriptFromGraphs(): string
 
         ArrayUtil::recursiveKeySort($data);
 
-        return '<script type="application/ld+json">'."\n".json_encode($data, JSON_PRETTY_PRINT | JSON_UNESCAPED_UNICODE)."\n".'</script>';
+        $return = [];
+
+        foreach ($data as $context) {
+            $return[] = sprintf(
```

Comment:
Can we add a comment that we only do this because Safari does not support one array-style JSON? Just so that we don't forget because if I look at this, I'm going "this is unnecessary" and I would probably change it back in a few years if there's no comment 🤣

Reply:
Added in 6a254e6cf1cc9943cf90b639eaa464248bb85924.